editor/question-ctrl.js

```diff
@@ -26,6 +26,7 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
 	$scope.questionData = {'max_score': 1};
 	$scope.questionData.isShuffleOption = false;
 	$scope.questionData.isPartialScore = true;
+	$scope.questionData.eval_unordered = false;
```

Comment:
Define all these in inside object declaration of $scope.questionData = ...,

$scope.questionData = {'max_score': 1, 'isShuffleOption': false, ....};